app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @user = current_user
+    @like = Like.new(author_id: @user.id, post_id: @post.id)
+
+    if @like.save
+      @post = Post.find(params[:post_id])
+      redirect_to user_post_path(user_id: @post.author_id, id: @post.id)
+    else
+      redirect_to user_post_path(user_id: @post.author_id, id: @post.id),
+                  alert: 'An error has occurred while creating the like'
+    end
+  end
```

Comment:
✅ A user is able to add a like to posts 👍🏻 

Kindly refactor the create action to

```ruby
def create
      @post = Post.find(params[:post_id])
      @like = @current_user.likes.new(post_id: @post.id)

      respond_to do |format|
        if @like.save
          format.html { redirect_to @post }
        else
          format.html { redirect_to @post }
        end
      end
    end
```